pygmt/helpers/tempfile.py

```diff
@@ -84,7 +84,7 @@ def read(self, keep_tabs=False):
         content : str
             Content of the temporary file as a Unicode string.
         """
-        with open(self.name) as tmpfile:
+        with open(self.name, mode="r", encoding="utf8") as tmpfile:
```

Comment:
I'm getting a little confused with understanding [PEP597](https://www.python.org/dev/peps/pep-0597/) (which is meant to be added in Python 3.10). Is `encoding="utf8"` recommended? Or should `encoding="locale"` be used instead?

Reply:
As I understand it, `encoding="locale"` is only available after Python 3.10.

Reply:
Ah ok, then utf8 is good then since we're supporting Python 3.7 and above.